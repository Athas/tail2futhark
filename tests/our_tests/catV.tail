let v0:[int]1 = [1,2,3] in
let v1:[int]1 = [4,5,6] in
let v2:[int]0 = reduce{[int],[0]}(addi, 0, cat(v0,v1)) in
i2d(v2)
