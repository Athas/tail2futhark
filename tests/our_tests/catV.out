fun real main() =
  let t_v0 = [1,2,3] in
  let t_v1 = [4,5,6] in
  let t_v2 = reduce(op +,0,concat(t_v0,t_v1)) in
  toReal(t_v2)